index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./styles.css">
+    <!-- font awesome -->
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.3.0/css/all.min.css" integrity="sha512-SzlrxWUlpfuzQ+pcUCosxcglQRNAq/DZjVsC0lE40xsADsfeQoEypE+enwcOiGjk/bSuGGKHEyjSoQ1zVisanQ==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <title>Youtube Clone</title>
+</head>
+<body>
+    <header class="header">
+        <nav>
+            <ul class="navbar">
+                <li>
+                    <i class="fa-solid fa-bars menu-icon icons"></i>
+                </li>
+                <li class="youtube-logo">
+                    <img src="./images/icon.png" class="logo" alt="">
+                    <h2>YouTube</h2>
+                </li>
+                <li class="search-bar">
+                    <input type="text" placeholder="Search" class="search-text">
+                    <i class="fa-solid fa-magnifying-glass search-glass icons"></i>
+                </li>
+                <li class="mic">
+                    <i class="fa-solid fa-microphone icons"></i>
+                </li>
+                <li>
+                    <i class="fa-solid fa-video icons"></i>
+                </li>
+                <li>
+                    <i class="fa-regular fa-bell icons"></i>
+                </li>
+                <li class="profile">
+                    <img src="./images/profile-pic.png" class="profile" alt="">
+                </li>
+            </ul>
+        </nav>
+    </header>
+    <main>
+        <section class="container">
+            <div class="video">
+                <!-- video goes here -->
```

Comment:
I think it would be nice to practice the use of descriptive comments rather than action comments. Descriptive comments describe what is enclosed in a certain section.